app/database_etl/airtable_records_handler/airtable_records_formatter.py

```diff
@@ -141,6 +142,14 @@ def add_test_adjustments(df: pd.DataFrame) -> pd.DataFrame:
     # Get all unique airtable_record_ids that are new/have been modified
     new_airtable_record_ids = diff['airtable_record_id'].unique()
 
+    # Add all unique airtable_record_ids for which test adjustment was unsuccessful 
```

Comment:
would we eventually want to hide this behind a flag or something

Reply:
Possible -- maybe behind a switch that we can throw when we call the ETL pipeline? 

For now, I suspect it's mostly for local use, and just valuable to have here commented out for folks to use locally as needed.